src/zope/app/form/browser/__init__.py

```diff
@@ -17,67 +17,66 @@
 # the implementation of widgets has moved to zope.formlib.widgets
 # import directly from there instead.
 
-from zope.formlib.widget import BrowserWidget, DisplayWidget
+from zope.formlib.widget import BrowserWidget
+from zope.formlib.widget import DisplayWidget
 from zope.formlib.widget import UnicodeDisplayWidget
-
-from zope.formlib.widgets import TextWidget, BytesWidget
-from zope.formlib.widgets import TextAreaWidget, BytesAreaWidget
-from zope.formlib.widgets import PasswordWidget, FileWidget
-from zope.formlib.widgets import ASCIIWidget, ASCIIAreaWidget
-from zope.formlib.widgets import IntWidget, FloatWidget
-from zope.formlib.widgets import DecimalWidget
-from zope.formlib.widgets import DatetimeWidget, DateWidget
-from zope.formlib.widgets import DatetimeI18nWidget
-from zope.formlib.widgets import DateI18nWidget
-from zope.formlib.widgets import DatetimeDisplayWidget
-from zope.formlib.widgets import DateDisplayWidget
-from zope.formlib.widgets import BytesDisplayWidget
-from zope.formlib.widgets import ASCIIDisplayWidget
-from zope.formlib.widgets import URIDisplayWidget
-
+# Widgets that let you enter several items in a sequence
+# These widgets are multi-views on (sequence type, value type)
+# Widgets that let you choose several items from a list
+# These widgets are multi-views on (field, vocabulary)
+# Widgets that let you choose a single item from a list
+# These widgets are multi-views on (field, vocabulary)
```

Comment:
Unfortunately `isort` (I guess) move the comments around and now they are meaningless, as they are out of context 😕

Reply:
I removed the comment.